docs/src/main/paradox/discovery.md

```diff
@@ -179,16 +179,25 @@ spec:
 ## Discovery Method: AWS API - EC2 Tag-Based Discovery
 
 If you're an AWS user, you can use tags to simply mark the instances that belong to the same cluster. Use a tag that
-has "service" as the key and set the value to be the name of your service. Note that this implementation is adequate
-for users running service clusters on *vanilla* EC2 instances. If you're using Amazon EKS (Amazon Elastic Container 
-Service for Kubernetes) or plain Amazon ECS, then this is not the right choice and you may want to use 
-the @ref:['Kubernetes API'-based discovery method](discovery.md#discovery-method-kubernetes-api) or some other solution.
-
+has "service" as the key and set the value to be the name of your service in `application.conf`.
```

Comment:
Maybe also mention the key in application.conf where the service name has to be `akka.cluster.bootstrap.contact-point-discovery.service-name` ?

Reply:
Done. Let me know what you think.